Added section on caching PAT (#6405)
github/docs: content/github/authenticating-to-github/creating-a-personal-access-token.md

If you are not prompted for your username and password, your credentials may be cached on your computer. You can [update your credentials in the Keychain](/github/getting-started-with-github/updating-credentials-from-the-macos-keychain) to replace your old password with the token.

> ### Caching a token with git
> 

Reviewer: Caching a token is part of using the token on the command line, so let's add this new content to the existing section instead of creating a new one.

```suggestion
```

> Rather than manually entering your personal access token every time you perform an HTTPS Git operation, you can cache your token with a Git client. Git will temporarily store your credentials in memory until an expiry interval has passed. You can also store the token in a plain text file that Git can read before every request. Information on caching your personal access token can be found in [Caching your GitHub credentials in Git](https://docs.github.com/en/github/getting-started-with-github/caching-your-github-credentials-in-git).

Reviewer: ```suggestion
Instead of manually entering your PAT for every HTTPS Git operation, you can cache your token with a Git client. Git will temporarily store your credentials in memory until an expiry interval has passed. You can also store the token in a plain text file that Git can read before every request. For more information, see  "[Caching your {% data variables.product.prodname_dotcom %} credentials in Git](/github/getting-started-with-github/caching-your-github-credentials-in-git)."
```

If you are not prompted for your username and password, your credentials may be cached on your computer. You can [update your credentials in the Keychain](/github/getting-started-with-github/updating-credentials-from-the-macos-keychain) to replace your old password with the token.

> Instead of manually entering your PAT for every HTTPS Git operation, you can cache your token with a Git client. Git will temporarily store your credentials in memory until an expiry interval has passed. You can also store the token in a plain text file that Git can read before every request. For more information, see "[Caching your {% data variables.product.prodname_dotcom %} credentials in Git](/github/getting-started-with-github/caching-your-github-credentials-in-git)."

Reviewer: ```suggestion
Instead of manually entering your PAT for every HTTPS Git operation, you can cache your PAT with a Git client. Git will temporarily store your credentials in memory until an expiry interval has passed. You can also store the token in a plain text file that Git can read before every request. For more information, see "[Caching your {% data variables.product.prodname_dotcom %} credentials in Git](/github/getting-started-with-github/caching-your-github-credentials-in-git)."
```
